tests/conftest.py

```diff
@@ -14,13 +16,64 @@ def remove_headers(response: dict):
         response["headers"] = {}
         return response
 
+    def scrub_requests(request: dict):
```

Comment:
Is `request` really a `dict`? I don't think you can do `request.body` on a Python dict. 👀